source/modules/collection-api/event/actions/update.md.erb

```diff
@@ -1,27 +1,22 @@
 ### Event Updates
-
-```php
-<?php
-
-use Inspetor;
-
-$inspetor = $this->getConfiguredInspetor();
-
-$inspetor->trackEventUpdate($event);
-
-?>
-```
-```python 
-import inspetor
-
-inspetor = self.getConfiguredInspetor()
-
-inspetor.track_event_update(event)
-
+```sh
+curl -XPUT https://collection.inspcdn.net/events \
+    -d '{
+        "id":          "12345",
+        "name":        "Reveillon 2021",
+        "sessions":  [
```

Comment:
Should be `event_datetimes`

Reply:
changed to `dates` per service code https://github.com/inspetor/http-collection-service/blob/master/functions/schemas/event.py#L30